elastic/kibana
# [Advanced settings] Set order for category panels

## Summary

This PR sets the order for the category panels so that it matches the order in the original Advanced settings page.

### How to test
1. Start Es with `yarn es serverless` and Kibana with `yarn serverless-{es/oblt/security}`
2. Go to Management -> Advanced Settings
3. Verify that the order of the categories is the same as the category order in the existing self-managed Advanced settings page (as of 8.10). Keep in mind that some of the categories are missing in serverless because some settings have been filtered out.
